base/post_hook_rocm_smi_lib.cmake

```diff
@@ -1,6 +1,15 @@
-target_include_directories(rocm_smi64 PUBLIC
-  "$<INSTALL_INTERFACE:lib/rocm_sysdeps/include>"
-)
-target_include_directories(oam PUBLIC
-  "$<INSTALL_INTERFACE:lib/rocm_sysdeps/include>"
-)
+# smi_lib public headers include DRM, so in the bundled case, we must ensure
```

Comment:
I would find it useful in at least one of these places to spell out "Direct Rendering Manager" for DRM (assume that's what this is). I'm so used to it meaning Digital Rights Management from video/graphics/display technology.

Reply:
Changed it to libdrm. I can't bring myself to type the full name (and didn't even know that's what it stood for... even though I have spent a lot of up close and personal time with libdrm).